oct/ansible/oct/roles/docker/tasks/configure_docker_daemon_storage.yml

```diff
@@ -1,40 +1,18 @@
 ---
-- name: create a logical volume for Docker data
-  lvol:
-    vg: '{{ origin_ci_docker_volume_group }}'
-    lv: 'docker-data'
-    size: '70%FREE'
```

Comment:
i don't see a good reason for us not to allocate the entire VG for our use case.